docs/streams/developer-guide/config-streams.html

```diff
@@ -284,6 +284,11 @@ <h4><a class="toc-backref" href="#id5">bootstrap.servers</a><a class="headerlink
             <td colspan="2">Timestamp extractor class that implements the <code class="docutils literal"><span class="pre">TimestampExtractor</span></code> interface.</td>
             <td>See <a class="reference internal" href="#streams-developer-guide-timestamp-extractor"><span class="std std-ref">Timestamp Extractor</span></a></td>
           </tr>
+          <tr class="row-odd"><td>upgrade.from</td>
```

Comment:
This should be `row-even` and all the row below need to be "switched" (an annoying task... We really need to move off HTML...)

Reply:
Oh I totally missed that...wow that's awful 😬

Reply:
What difference does it make? I notice it's also screwed up between `partition.grouper` and `processing.guarantee`, and between `replication.factor` and `retries` and `retry.backoff.ms`

Reply:
Well, I guess we should fix the screw up than? It should influence the coloring if the row from my understanding.

Reply:
Locking at https://kafka.apache.org/documentation/streams/developer-guide/config-streams.html#optional-configuration-parameters it does actually not influence the colering... Not sure why it's there now... \cc @JimGalasyn

Reply:
Hm. Interestingly,[ the table](https://kafka.apache.org/24/documentation/streams/developer-guide/config-streams.html#optional-configuration-parameters) seems to render correctly nevertheless...I can make a ticket for it, just trying to understand what (if anything) it actually affects

Reply:
Whoops, jinx

Reply:
Maybe @londoncalling can help on the even/odd pattern?

Reply:
FWIW I did fix it in this PR, and have a followup PR to fix it in 2.4-trunk: https://github.com/apache/kafka/pull/7837

If maintaining this pattern really is important, we should still fix the other configs in separate PR(s) so we can target the right branches